.github/workflows/tests.yml

```
@@ -1,16 +1,30 @@
+  pull_request:
+
+env:
+  HUSKY: '0'
 
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@master
-      - uses: actions/setup-node@v1
+      - uses: actions/checkout@v3
```

actionlint (1.7.10)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)